### PR TITLE
Contritubing Documentation 

### PR DESCRIPTION
**Pull request by wandapeter**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ to jorge.castro@gmail.com
 
 ## I Have a Question
 
-> If you want to ask a question, ask in the [discussion forum](https://github.com/orgs/ublue-os/discussions)
+> If you want to ask a question, ask in the [Issues](https://github.com/secureblue/secureblue/issues).
```
Comment (RoyalOughtness) [inline CONTRIBUTING.md]: ask in [Issues]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -86,21 +86,77 @@ Here's an example: https://github.com/ublue-os/nvidia/pull/49
...
     
-    podman image ls 
+- Upload image to the container registry
 
-TODO: Set up and push to your own local registry
+```
+podman push --tls-verify=false registry.dev.local:5000/ublue-main:dev-latest
+```
+
+### Rebase to DEV image
```
Comment (RoyalOughtness) [inline CONTRIBUTING.md]: all of these changes should be done upstream if the upstream documentation is inaccurate

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -86,21 +86,77 @@ Here's an example: https://github.com/ublue-os/nvidia/pull/49
...
+insecure = true
+```
+
+### Clone the secureblue repo
+
+    git clone https://github.com/secureblue/secureblue
 
 ### Build the image
     
+First makre sure to edit select the Image you would like to build by editing the `RECIPE` Argument in the Containerfile. Here is an example selecting Silverblue Main:
```
Comment (RoyalOughtness) [inline CONTRIBUTING.md]: make

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

**Comment by RoyalOughtness** [comment]
The only blocking issue here is that all edits to upstream-relevant documentation should be submitted upstream: https://github.com/ublue-os/startingpoint

Please remove those changes from this PR and it can be merged then.

**Comment by wandapeter** [comment]
makes perfect sense. I'll submit the changes to CONTRIBUTING.md upstream

**Comment by RoyalOughtness** [comment]
> makes perfect sense. I'll submit the changes to CONTRIBUTING.md upstream

There are still changes here that are relevant though. like changing the name to secureblue, adding to the readme, and linking to the issues page instead of the discussion page. No need to close the PR :) those changes should still be merged here

**Comment by wandapeter** [comment]
Removed all the changes in the "Building locally". Left the title and link to issuses.  I'll create a PR with the updated documentation upstream. :-)

**Pull request merged**